### DISCUSSION TITLE
[Minor issue] Hovering over map success rate in lazer browser does not give the pass/fail ratio

### DISCUSSION THREAD

**Discussion by FunnyFortniteFan** [ppy/osu#17415] [Q&A] [answered]
Issue: When you hover over the success rate in the lazer web browser, it does not give the pass fail ratio
![image](https://user-images.githubusercontent.com/71988906/159625647-77d549ff-9d8d-4dd4-b908-0511db9eaa17.png)

On the base website, I.E just going onto osu.ppy.sh thru google chrome or whatever, when hovering over success rate it gives the pass/fail ratio.
![image](https://user-images.githubusercontent.com/71988906/159625450-084a1d7e-199a-4d90-a492-f218e988ceeb.png)

Hardware: Windows 10, nvidia 1660 super

**Comment by frenzibyte** [comment, marked as answer]
Will be resolved by #17613.

**Reply by FunnyFortniteFan** [reply]
thank you!!
